v3/src/components/case-table/case-table.scss

```diff
@@ -14,6 +16,7 @@
         .codap-column-header-content {
           width: 100%;
           height: 100%;
+          font-weight: bold !important;
```

Comment:
Is this for the drag overlay? We are already specifying `fontWeight="bold"` in inline styling.

Reply:
Good catch -- probably redundant. I'll check and remove it if so.